src/components/CurrentRefinements/__tests__/CurrentRefinements-test.tsx

```diff
@@ -1,7 +1,9 @@
-import React from 'react';
+/** @jsx h */
```

Comment:
can we set this globally instead of per file, or is the problem that the import will disappear or be "unused" otherwise?

Reply:
At first I wanted to avoid having to declare this pragma everywhere but VS code was unaware of the pragma when it was not present in the file and thus raised errors. We can remove it later with Babel plugins if it's too annoying.